version/1/4/it/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Codice di Condotta Convenuto del Collaboratore
```

Comment:
In questo caso mi ero basato sul titolo della versione precedente, ma credevo anch'io che si potesse migliorare.
Grazie per le altre correzioni, a breve vedo di inserirle! 👍